.travis.yml

```diff
@@ -31,13 +31,17 @@ install:
 env:
   - KUBECONFIG=${HOME}/.kube/config PATH=$HOME/k8scli:${PATH}
 
-script:
+before_script:
   - cd ${TRAVIS_BUILD_DIR}
-  - glide install -v
-  - ./fission-bundle/build.sh
   - hack/verify-gofmt.sh
   - hack/verify-govet.sh
+  - glide install -v
+  - go build -o fission/fission fission/*.go
+  - ./fission-bundle/build.sh
   - hack/runtests.sh
+
+script:
+  - cd ${TRAVIS_BUILD_DIR}
```

Comment:
You probably tested this, but just to be sure: do you need to change the current directory again in the script stage? Doesn't this propagate from the `before_script`?

Reply:
Not sure will the path propagate from the previous stage but looks like it works. 
I think another advantage is that it helps people to know the working directory to `${TRAVIS_BUILD_DIR}` in current stage without checking the configs in previous stages.